src/uu/factor/src/numeric/gcd.rs

```diff
@@ -9,39 +9,43 @@
 use std::cmp::min;
 use std::mem::swap;
 
-pub fn gcd(mut n: u64, mut m: u64) -> u64 {
+pub fn gcd(mut u: u64, mut v: u64) -> u64 {
     // Stein's binary GCD algorithm
     // Base cases: gcd(n, 0) = gcd(0, n) = n
-    if n == 0 {
-        return m;
-    } else if m == 0 {
-        return n;
+    if u == 0 {
+        return v;
+    } else if v == 0 {
+        return u;
```

Comment:
```suggestion
    if u == 0 {
        return v;
    }
    if v == 0 {
        return u;
```

Reply:
Any particular reason this is the preferred form?

Reply:
My Mozilla habits, the "else" is useless and makes it a tiny bit harder to read

Reply:
*shrugs.* I find the reverse a tiny bit harder to read, but I also don't care to argue >_>'
Done